packages/nodejs/src/interfaces/span_processor.ts

```diff
@@ -0,0 +1,44 @@
+import { HrTime, Span, SpanAttributes, SpanContext } from "@opentelemetry/api"
+
+/**
+ * OpenTelemetrySpanProcessor is based on OpenTelemetry's internal span processor.
+ */
+export interface OpenTelemetrySpanProcessor {
```

Comment:
Instead of copying the types, I believe it should be possible to add the package with the types as a dev dependency and use `import type` to import only the type definitions.

Reply:
@unflxw not sure what you mean here. If it's a dev dependency how can we use it once the package is shipped?